baked_in.go

```diff
@@ -188,6 +189,7 @@ var (
 		"iso3166_1_alpha2":        isIso3166Alpha2,
 		"iso3166_1_alpha3":        isIso3166Alpha3,
 		"iso3166_1_alpha_numeric": isIso3166AlphaNumeric,
+		"language_tag":            isLanguageTag,
```

Comment:
@joshuaslate what do you think about changing this to `bcp47_language_tag` to be specific and future proof for adding others?

Reply:
Thank you for the review, @deankarn. Good call-- I've updated the PR 😄